src/Integration.Vsix.UnitTests/CFamily/CMake/CompilationDatabaseRequestTests.cs

```diff
@@ -38,23 +38,31 @@ public class CompilationDatabaseRequestTests
     {
         private readonly RequestContext ValidContext = new RequestContext("cpp", Mock.Of<ICFamilyRulesConfig>(), "file.txt", "pchFile.txt", null);
         private readonly CompilationDatabaseEntry ValidDbEntry = new CompilationDatabaseEntry { File = "file.txt", Directory = "c:\\", Command = "a command" };
+        private readonly IReadOnlyDictionary<string, string> ValidEnvVars = new Dictionary<string, string> { { "key1", "value1" } };
 
         [TestMethod]
-        public void Ctor_NullArguments_Throws()
+        public void Ctor_InvalidArguments_Throws()
         {
-            Action act = () => new CompilationDatabaseRequest(null, ValidContext);
+            Action act = () => new CompilationDatabaseRequest(null, ValidContext, ValidEnvVars);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("databaseEntry");
 
-            act = () => new CompilationDatabaseRequest(ValidDbEntry, null);
+            act = () => new CompilationDatabaseRequest(ValidDbEntry, null, ValidEnvVars);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("context");
         }
 
+        [TestMethod]
+        public void Ctor_NullEnvVars_DoesNotThrow()
+        {
+            var testSubject = new CompilationDatabaseRequest(ValidDbEntry, ValidContext, null);
+            testSubject.EnvironmentVariables.Should().BeNull();
+        }
+
         [TestMethod]
         [DataRow(null, "args")]
         [DataRow("", "args")]
         [DataRow("cmd", null)]
         [DataRow("cmd", "")]
-        public void Ctor_ValidDBEntry_ShouldNotThrow(string command, string args)
+        public void Ctor_ValidCommandArgsCombination_ShouldNotThrow(string command, string args)
```

Comment:
In retrospect, I didn't need to rename this test or the next one. However, the new names are probably more descriptive of what the test is doing.